dom-bom/index.js

```diff
@@ -0,0 +1,63 @@
+const buttonFind = document.querySelector('.button-find');
+const buttonMyWeather = document.querySelector('.button-myWeather');
+const buttonClear = document.querySelector('.button-clear');
+const tableHead = `<tr><th>City</th><th>Country</th><th>t, °C</th><th>t, °F</th></tr>`
+
+function getWeatherByLocation() {
+
+    navigator.geolocation.getCurrentPosition (successF, errorF);
+    function successF (position) {
+        let inputField = document.getElementsByClassName('input-class')[0];
+        inputField.value = ''; //clear the input field when we're searching for weather by our location
+        let lat = position.coords.latitude;
+        let lng = position.coords.longitude;
+        let value = lat + ', ' + lng;
+        getWeatherByNameOfCity(value);
+    }
+    function errorF () {
+        alert ('Your location unavailable');
+    }
+}
+
+function getWeatherByNameOfCity(currentLocation) {
+
+    let ourInput = document.getElementsByClassName('input-class')[0].value;

```

Comment:
Instead of get the output element on every search firing, it's better to get it outside of this function

Reply:
but this way I'll can't get the value because it'll be empty always